lib/types/types.d.ts

```diff
@@ -89,7 +89,13 @@ export type UserRemovedData = {
     channel_id?: string;
     remover_id: string;
 } & BaseData;
-export type WebsocketEventData = EmptyData | HelloData | CallStartData | UserDisconnectedData | UserConnectedData | UserMutedUnmutedData | UserVoiceOnOffData | UserScreenOnOffData | UserRaiseUnraiseHandData | EmojiData | UserReactionData | CallHostChangedData | CallRecordingStateData | UserState | UserDismissedNotification | CallStateData | JobStopData | UserRemovedData;
+export type LiveCaptionData = {
+    channelID: string;
```

Comment:
If it's not too painful I'd start using `channel_id` for new types to eventually reach case consistency.

Reply:
Thanks, changed.